### ISSUE TITLE
Misc. bug:  3 time increase in t/s if Mullvad 15.0.7 Browser used. Compared to Brave 1.88.127 (Official Build) (64-bit)

### ISSUE THREAD

**Issue by gogich77** [ggml-org/llama.cpp#20485]
### Name and Version

workspace/llama.cpp/build/bin/llama-server --version                                                                            
ggml_cuda_init: found 3 ROCm devices (Total VRAM: 158492 MiB):
  Device 0: AMD Radeon Graphics, gfx1201 (0x1201), VMM: no, Wave Size: 32, VRAM: 32624 MiB (32536 MiB free)
  Device 1: AMD Radeon Graphics, gfx1201 (0x1201), VMM: no, Wave Size: 32, VRAM: 32624 MiB (32536 MiB free)
  Device 2: AMD Ryzen 9 9950X 16-Core Processor, gfx1036 (0x1036), VMM: no, Wave Size: 32, VRAM: 93244 MiB (93240 MiB free)
version: 8324 (983df142a)
built with GNU 13.3.0 for Linux x86_64

### Operating systems

Linux

### Which llama.cpp modules do you know to be affected?

llama-server

### Command line

```shell
/workspace/llama.cpp/build/bin/llama-server -m /data2/llm/models/Qwen3.5-27B-UD-Q8_K_XL.gguf  --mmproj /data2/llm/models/Qwen3.5-27B-UD-Q8_K_XL-mmproj-F16.gguf   --device ROCm0,ROCm1 --split-mode layer --port 8000 --host 0.0.0.0 --fit on --jinja  -b 512 -ub 512     --ctx-size 32768     --temp 1.0     --top-p 0.95     --top-k 20     --min-p 0.00     --chat-template-kwargs '{"enable_thinking":false}'
```

### Problem description & steps to reproduce

2 browsers 

**Brave 1.88.127 (Official Build) (64-bit) Chromium: 146.0.7680.71**  -->  _4.50 t/s_

 **Mullvad browser 15.0.7 based on Mozila Firefox 140.8.0esr**  -->  _16.07 t/s_

in the 2 the query is HI 

### First Bad Commit

First time see it.

### Relevant log output

<details>
<summary>Logs</summary>
#################
Brave 1.88.127 (Official Build) (64-bit)
Chromium: 146.0.7680.71
##################
Hello! How can I help you today?

Qwen3.5
27B
Reading
Generation
10 tokens
2.2s
4.50 t/s

#################################################################################################

srv          init: init: chat template, thinking = 1
main: model loaded
main: server is listening on http://0.0.0.0:8000
main: starting the main loop...
srv  update_slots: all slots are idle
srv  params_from_: Chat format: peg-native
slot get_availabl: id  3 | task -1 | selected slot by LRU, t_last = -1
slot launch_slot_: id  3 | task -1 | sampler chain: logits -> ?penalties -> ?dry -> ?top-n-sigma -> top-k -> ?typical -> top-p -> ?min-p -> ?xtc -> ?temp-ext -> dist 
slot launch_slot_: id  3 | task 0 | processing task, is_child = 0
slot update_slots: id  3 | task 0 | new prompt, n_ctx_slot = 32768, n_keep = 0, task.n_tokens = 13
slot update_slots: id  3 | task 0 | n_tokens = 0, memory_seq_rm [0, end)
slot update_slots: id  3 | task 0 | prompt processing progress, n_tokens = 9, batch.n_tokens = 9, progress = 0.692308
srv  log_server_r: done request: POST /v1/chat/completions 127.0.0.1 200
slot update_slots: id  3 | task 0 | n_tokens = 9, memory_seq_rm [9, end)
slot init_sampler: id  3 | task 0 | init sampler, took 0.01 ms, tokens: text = 13, total = 13
slot update_slots: id  3 | task 0 | prompt processing done, n_tokens = 13, batch.n_tokens = 4
slot print_timing: id  3 | task 0 | 
prompt eval time =     936.61 ms /    13 tokens (   72.05 ms per token,    13.88 tokens per second)
       eval time =    2224.18 ms /    10 tokens (  222.42 ms per token,     4.50 tokens per second)
      total time =    3160.78 ms /    23 tokens
slot      release: id  3 | task 0 | stop processing: n_tokens = 22, truncated = 0
srv  update_slots: all slots are idle

^Csrv    operator(): operator(): cleaning up before exit...
llama_memory_breakdown_print: | memory breakdown [MiB] | total    free     self   model   context   compute    unaccounted |
llama_memory_breakdown_print: |   - ROCm0 (Graphics)   | 32624 = 15132 + (15955 = 14147 +    1335 +     472) +        1536 |
llama_memory_breakdown_print: |   - ROCm1 (Graphics)   | 32624 = 12952 + (19273 = 17299 +    1310 +     663) +         398 |
llama_memory_breakdown_print: |   - Host               |                   2701 =  2425 +       0 +     276                |
root@biggogo:/workspace/llama.cpp/build/bin# 

#####################################
Mullvad browser 15.0.7 based on Mozila Firefox 140.8.0esr
#####################################
Hello! How can I help you today?
10 tokens 0.6s 16.07 t/s

#####################################

main: server is listening on http://0.0.0.0:8000
main: starting the main loop...
srv  update_slots: all slots are idle
srv  params_from_: Chat format: peg-native
slot get_availabl: id  3 | task -1 | selected slot by LRU, t_last = -1
slot launch_slot_: id  3 | task -1 | sampler chain: logits -> ?penalties -> ?dry -> ?top-n-sigma -> top-k -> ?typical -> top-p -> ?min-p -> ?xtc -> ?temp-ext -> dist 
slot launch_slot_: id  3 | task 0 | processing task, is_child = 0
slot update_slots: id  3 | task 0 | new prompt, n_ctx_slot = 32768, n_keep = 0, task.n_tokens = 13
slot update_slots: id  3 | task 0 | n_tokens = 0, memory_seq_rm [0, end)
slot update_slots: id  3 | task 0 | prompt processing progress, n_tokens = 9, batch.n_tokens = 9, progress = 0.692308
srv  log_server_r: done request: POST /v1/chat/completions 127.0.0.1 200
slot update_slots: id  3 | task 0 | n_tokens = 9, memory_seq_rm [9, end)
slot init_sampler: id  3 | task 0 | init sampler, took 0.00 ms, tokens: text = 13, total = 13
slot update_slots: id  3 | task 0 | prompt processing done, n_tokens = 13, batch.n_tokens = 4
slot print_timing: id  3 | task 0 | 
prompt eval time =     705.92 ms /    13 tokens (   54.30 ms per token,    18.42 tokens per second)
       eval time =     622.16 ms /    10 tokens (   62.22 ms per token,    16.07 tokens per second)
      total time =    1328.08 ms /    23 tokens
slot      release: id  3 | task 0 | stop processing: n_tokens = 22, truncated = 0
srv  update_slots: all slots are idle
^Csrv    operator(): operator(): cleaning up before exit...
llama_memory_breakdown_print: | memory breakdown [MiB] | total    free     self   model   context   compute    unaccounted |
llama_memory_breakdown_print: |   - ROCm0 (Graphics)   | 32624 = 15132 + (15955 = 14147 +    1335 +     472) +        1536 |
llama_memory_breakdown_print: |   - ROCm1 (Graphics)   | 32624 = 12952 + (19273 = 17299 +    1310 +     663) +         398 |
llama_memory_breakdown_print: |   - Host               |                   2701 =  2425 +       0 +     276                |
root@biggogo:/workspace/llama.cpp/build/bin# 

```console

```
</details>

**Comment by gogich77** [comment]
Most probably a defect in Brave Browser. 

Another LLM

/workspace/llama.cpp/build/bin/llama-server -m /data2/llm/models/Qwen3-Coder-Next-UD-Q4_K_XL.gguf  --device ROCm0,ROCm1 --split-mode layer -c 90000 --port 8000 --host 0.0.0.0 --fit on --seed 3407 --temp 1.0 --top-p 0.95 --min-p 0.01 --top-k 40 -b 512 -ub 512 --jinja --alias quen3-coder  --repeat-penalty 1.05


**Brave (20+ tabs open)**
```
prompt eval time =     221.25 ms /     9 tokens (   24.58 ms per token,    40.68 tokens per second)
       eval time =    1310.45 ms /    12 tokens (  109.20 ms per token,     9.16 tokens per second)
      total time =    1531.71 ms /    21 tokens
```

**Mullvad ( single tab )**
```
prompt eval time =      67.58 ms /     9 tokens (    7.51 ms per token,   133.18 tokens per second)
       eval time =     293.63 ms /    12 tokens (   24.47 ms per token,    40.87 tokens per second)
      total time =     361.21 ms /    21 tokens
```

**Comment by gogich77** [comment]
It looks the client is causing a delay on the server. 
Maybe you can add it in your statistics
